.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,14 +1,22 @@
+## Scope and purpose
+
+Add a few words about why this PR is needed and what is its scope.
+
+Add mentions of things that are not covered here and are planed to be done
+in separate PRs.
+
+
 ## Remove this section
 
 Please have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
 
 Please note that the trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small; if you don't do them, it's likely that nobody will even notice your PR needs a review.
 
+
 ## Contributor Checklist:
 
 * [ ] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/<!-- Create a new one at https://twistedmatrix.com/trac/newticket and replace this comment with the ticket number. -->
 * [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
```

Comment:
```suggestion
```

I think it's worth removing this and just relying on CI here

Reply:
I do think it's useful to mention that command and link the coding standard. Maybe it doesn't have to be part of the checklist though.

The same applies for "additional checks" and "created a newsfragment": these are checked by CI, but it's good to have the links to the documentation immediately available.

Reply:
What does this change have to do with the release process at all?